Description: Remove the old template
 It was an old default that wasn't up-to-date. Since there was no
 customization, the best bet is to entirely remove it to avoid having the issue
 next time Sphinx changes something.
Author: Simon Chopin <chopin.simon@gmail.com>
Bug: http://github.com/pazz/alot/issues/503
diff --git a/docs/source/_templates/layout.html b/docs/source/_templates/layout.html
deleted file mode 100644
index b4384e4..0000000
--- a/docs/source/_templates/layout.html
+++ /dev/null
@@ -1,7 +0,0 @@
-{% extends "!layout.html" %}
-{% set script_files = [] %}
-
-{% block linktags %}{% endblock %}
-{% block relbar1 %}{% endblock %}
-{% block footer %}{% endblock %}
-{% block relbaritems %}{% endblock %}
